jdi-light/src/main/java/com/epam/jdi/light/elements/complex/table/BaseTable.java

```diff
@@ -142,7 +141,7 @@ public void setHeader(List<String> header) {
         this.header.setFinal(header);
     }
 
-    public JFunc1<String, String> SIMPLIFY = STRING_SIMPLIFY;
+    public JFunc1<String, String> SIMPLIFY = ELEMENT.simplifyString;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6397461)